bats/core/api/user.bats

```diff
@@ -24,3 +24,37 @@ setup_file() {
   language="$(graphql_output '.data.me.language')"
   [[ "$language" == "$new_language" ]] || exit 1
 }
+
+@test "support: default support message result" {
+  "skip"
```

Comment:
is this working? https://bats-core.readthedocs.io/en/stable/writing-tests.html#skip-easily-skip-tests

Reply:
yes locally when the OPENAI key is added.
it's skipped voluntarily on ci/cd because we don't want to pay for openai usage for testing. it's like twilio it's a pay per call